lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -156,7 +238,7 @@ export class StandardController implements IController {
     >;
 
     // Native Extension Provider
-    protected nativeExtensionProvider: NativeMessageHandler | undefined;
+    protected platformAuthProvider: NativeMessageHandler | undefined;
```

Comment:
This probably should be either `NativeMessageHandler` or `DOMHandler` correct?

Reply:
I haven't changed the type yet because I just wanted to test isPlatformAvailable() API e2e first. I was thinking of changing the implementation to `Map<String, NativeMessageHandler | DOMHandler>` like we discussed before but tied to Standard controller instead of being a global variable.